symbol_table/src/tests.rs

```diff
@@ -205,3 +224,78 @@ fn keep_vars() {
         assert_eq!(table.get(var), Some(&data));
     }
 }
+
+#[test]
+fn newing_stores_a_copy() {
+    let mut table = Table::new();
+    let var = var_m("var", &[]);
+
+    let level_zero = "zero".try_into().unwrap();
```

Comment:
Consider consolidating the Value initialization code. Grouping all the try_into().unwrap() boiler plate may be easier to read.